coforma/swift-tech-challenge
# Bug fix/Tweak typography and spacing for browse page

# Title
Bug fix/Tweak typography and spacing for browse page
# Summary

This updates the browse page to more closely align with what design has mocked up. Spacing has been added, typography updated, and attention was given to the mobile to desktop transition.

## Type of change

Please delete options that are not relevant.


# How To Test

1. run `yarn dev`, then navigate to `localhost:3000`
2. See the new styles!
3. Change the browser size to see the new tablet and mobile views.

# Checklist:
